### PR TITLE
chore: add code of conduct

### PR DESCRIPTION
**Pull request by machikoyasuda**
- removes Projects link from CoC cos we're not using it anymore

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the broader communities in which we participate.
+
+This code of conduct applies to all spaces managed by Reaction Commerce. This includes our [development chat room](https://gitter.im/reactioncommerce/reaction), [forums](https://forums.reactioncommerce.com), [blog](https://blog.reactioncommerce.com), mailing lists, [issue tracker](https://github.com/reactioncommerce/reaction/issues), [project boards](https://github.com/reactioncommerce/reaction/projects), Reaction events and meetups, and any other forums or service created by the core project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing <mailto:conduct@reactioncommerce.com>. For more details, please see our [Reporting Guidelines](https://docs.reactioncommerce.com/reaction-docs/master/reporting-guide).
```
Comment (mattt416) [inline CODE_OF_CONDUCT.md]: Do you need the `mailto:` in <mailto:conduct@reactioncommerce.com>?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors, and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the broader communities in which we participate.
+
+This code of conduct applies to all spaces managed by Reaction Commerce. This includes our [development chat room](https://gitter.im/reactioncommerce/reaction), [forums](https://forums.reactioncommerce.com), [blog](https://blog.reactioncommerce.com), mailing lists, [issue tracker](https://github.com/reactioncommerce/reaction/issues), [project boards](https://github.com/reactioncommerce/reaction/projects), Reaction events and meetups, and any other forums or service created by the core project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```
Comment (mattt416) [inline CODE_OF_CONDUCT.md]: Getting a 404 on `https://github.com/reactioncommerce/reaction/projects`, I'm not familiar projects but I'm wondering if this is private to team members?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (machikoyasuda) [reply CODE_OF_CONDUCT.md]: Yes.

Comment (mattt416) [reply CODE_OF_CONDUCT.md]: Did a quick test, it's not necessary (see [this](https://gist.github.com/mattt416/bc235f08e1ac5c1caedabbbf2857acf9)).  If you add the mailto you see the mailto in the actual HTML rendered string.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (zenweasel) [reply CODE_OF_CONDUCT.md]: I am also getting a 404 on Projects. I think we have just turned that off as it's something we don't use any more.

**Comment by machikoyasuda** [comment]
@zenweasel Removed project links from COC.

**Pull request merged**